nusantara/utils/common_parser.py

```diff
@@ -15,3 +19,102 @@ def load_conll_data(file_path):
             sentence = []
             seq_label = []
     return dataset
+
+
+def load_ud_data(filepath):
+    """
+    Load and parse conllu data.
+
+    Proposed by @fhudi for issue #34 and #9.
+
+    :param filepath: file path
+    :return: generator with schema following CONLLU
+    """
+    dataset_raw = parse(open(filepath).read())
+    return map(lambda sent: {**sent.metadata, **pd.DataFrame(sent).to_dict(orient="list")}, dataset_raw)
+
+
+def load_ud_data_as_nusantara_kb(filepath):
+    """
+    Load and parse conllu data, followed by mapping its elements to Nusantara Knowledge Base schema.
+
+    Proposed by @fhudi for issue #34 and #9.
+
+    :param filepath: file path
+    :return: generator for Nusantara KB schema
+    """
+    dataset_source = list(load_ud_data(filepath))
+
+    def as_nusa_kb(tokens):
+        sent_id = tokens["sent_id"]
+        offsets = get_span_offsets(tokens["form"], tokens["text"])
```

Comment:
Instead of `tokens["form"]`, we should use `tokens["text"]` instead, since it is the original string data before parsing.

Reply:
Sorry, can you elaborate on this?

Reply:
I just checked the update, it seems to work well. I think we can ignore this one then. Before, I am suggesting this one to ensure that there is no assumption on the number of spacing between each token.

Reply:
I see, coolbeans. Thanks